account_product_fiscal_classification/models/account_product_fiscal_classification_template.py

```diff
@@ -8,6 +8,7 @@
 class AccountProductFiscalClassificationTemplate(models.Model):
     _name = "account.product.fiscal.classification.template"
     _description = "Fiscal Classification Template"
+    _order = "sequence, id"
```

Comment:
```suggestion
    _order = "sequence, name"
```

if two items have the same sequence. (that occures, when updating this module), better to order by name.

Reply:
In the current implementation, there is no "_order", so it is order by id. So "sequence, id" keeps the current order on existing installations.

Reply:
indeed, but is it was we want ?

Reply:
yes ! That way, we don't change the habits of the users until they decide the exact order they want.

Reply:
I mean, order by id is wrong, and has been fixed by https://github.com/OCA/account-fiscal-rule/pull/275 monthes ago. 
But this patch has not been back-ported.
I think it should. 
the order by sequence is OK, but please take the opportunity to backport this trivial patch !

thanks !